src/models/generated/AVRTypeI.jl

```diff
@@ -34,7 +34,7 @@ Parameters of an Automatic Voltage Regulator Type I - Resembles IEEE Type DC1
 - `Te::Float64`: Field Circuit Time Constant in s, validation range: `(0, nothing)`
 - `Tf::Float64`: Stabilizer Time Constant in s, validation range: `(0, nothing)`
 - `Tr::Float64`: Voltage Measurement Time Constant in s, validation range: `(0, nothing)`
-- `Va_lim::NamedTuple{(:min, :max), Tuple{Float64, Float64}}`: Limits for pi controler `(Va_min, Va_max)`
+- `Va_lim::Min_Max`: Limits for pi controler `(Va_min, Va_max)`
```

Comment:
While we're breaking things, I find these aliases unpredictable. Could we standardize the naming conventions?
```
Min_Max -> MinMax
Up_Down -> UpDown
StartUp_ShutDown -> StartUpShutDown
From_To_Float -> FromToFloat
FromTo_ToFrom_Float -> I have no idea and don't see it being used
```

We could keep the originals to avoid breaking people - but change all uses.

Reply:
If you disagree, no worries, I'll still approve.

Reply:
I am fine with the rename, I will make the change and push a commit

Reply:
@daniel-thom I made the use of Alias consistent. I think that `FromTo_ToFrom` needs an exception because `FromToToFrom`is very confusing to read.